src/components/AddBook.jsx

```diff
@@ -52,7 +59,9 @@ const AddBook = () => {
           <option>Action</option>
           <option>Crime</option>
         </select>
-        <button type="submit">Add book</button>
+        <button type="submit" className={style.btn}>
+          ADD BOOK
+        </button>
       </form>
```

Comment:
- [ ] It would be nice if you could align the book input title field to the left so it's aligned with the book cards just as it is in the original Template.
- [ ] Also, the button should be aligned with the book cards on the right as well. Please work on that 👍 


**Your Design**

![image](https://user-images.githubusercontent.com/50941074/193430954-6c16c43f-7bfa-44cc-9290-679944f25b66.png)


**The Original Template**

![image](https://user-images.githubusercontent.com/50941074/193430984-0cbe6d0c-cfaa-4982-9af0-c3dfdc58092e.png)